airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/__init__.py

```diff
@@ -1,28 +1,3 @@
-"""
```

Comment:
I'm a bit late for this, but why are these removed instead of transformed into the # comments?